mlir/include/mlir/Dialect/MIOpen/MIOpen.h

```diff
@@ -83,14 +83,14 @@ namespace miopen {
 /// shape down or the lower shape up, such as passThrough() or pad(), "start" is
 /// the coordinate space you began building from and end is the coordinate space
 /// you're building towards (whether that's the upper or lower coordinates)
-class CoordTransformsBuilder {
+class TransformMapBuilder {
```

Comment:
Move to it's own file. Many things that include MIOpen do not need this.

Reply:
Yeah, that's a reasonable ask since we're already here

Reply:
Done